lsb_vsx/lsb_vsx-2.0-1-config/patches/05_defines.patch

```diff
@@ -0,0 +1,42 @@
+--- lsb_vsx-2.0-1/src/defines.mk	2022-07-18 12:39:27.910040986 +0200
++++ lsb_vsx-2.0-1-phoenix/src/defines.mk	2022-09-21 11:16:52.834131085 +0200
+@@ -88,10 +88,18 @@
+ DTET_CDEFS = -DINETD
+ 
+ # CC - the name of the C compiler
+-CC = gcc
```

Comment:
just change to `CC ?= gcc` - that way it will only be set if env value is not present.

That way - no need to specify by platform (this is a repetition from `build.sh`).

Also: don't use tabs (`\t`) for indentation in Makefiles, if necessary - use double space